package sample.java8.exception;

/**
 * 异常对象是否可以重用？不用每次抛出的时候都new
 */
public class TestReuseException {

}
